src/Lykke.AlgoStore.CSharp.AlgoTemplate.Models/Repositories/AlgoClientInstanceRepository.cs

```diff
@@ -70,6 +70,13 @@ public AlgoClientInstanceRepository(INoSQLTableStorage<AlgoClientInstanceEntity>
             return entitiy.ToModel();
         }
 
+        public async Task<AlgoClientInstanceData> GetAlgoInstanceDataByAuthTokenAsync(string authToken)
+        {
+            var entity = await _table.GetDataAsync(KeyGenerator.GenerateAuthTokenPartitionKey(authToken));
```

Comment:
we can pass the instance id in order to get only one row.

Reply:
There wouldn't be more than one row per auth token, and in most cases we wouldn't have the instance ID (since the idea was that the instance would only know of it's auth token)

Reply:
ok